charts/dify/templates/configmap.yaml

```
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "dify.fullname" . }}-sandbox-config
+data:
+  python-requirements.txt: |
+{{- range .Values.sandbox.config.python_requirements }}
+    {{ . }}
+{{- end }}
```

此处我提议直接

```suggestion
{{ .Values.sandbox.config.python_requirements | indent 4 }}
```

values 内让用户填写文件内容, 你觉得如何?  这样更容易理解, 也不容易出错
```
sandbox:
  config:
    python_requirements: |
      foo
      bar
```